vm.c

```diff
@@ -95,6 +96,9 @@ static inline void vm_push(vm_env *env, size_t n);
 /* OPCODE impl max size */
 #define OPCODE_IMPL_MAX_SIZE 256
 
+/* Label hash table size */
+#define LABEL_HASH_TABLE_SIZE 49157
```

Comment:
Rename to `LABEL_HT_SIZE`, which stands for the same but shorter.

Reply:
Got it.